### DISCUSSION TITLE
限定公開記事を公開したあとに限定公開に戻したい

### DISCUSSION THREAD

**Discussion by blono** [increments/qiita-discussions#15] [要望・アイデア・不具合]
現状、限定公開記事を公開した場合、そこでゲームオーバーだという認識です。
アドベントカレンダーとかで限定公開にしていた記事を誤って公開してしまったときとか、再度限定公開に戻す事ができないので不便に感じることがあります。

過去に書いた記事も内容が古くなってきたため限定公開にしたい（削除はしたくない）という運用がしたいですが、それができないのでちょっと不便に思いました。

**Comment by kskwtnk** [comment]
@blono 

ご意見ありがとうございます！

> 現状、限定公開記事を公開した場合、そこでゲームオーバーだという認識です。

こちら、私も経験があり、非常にわかります……😭

限定公開に戻したいという声は一定数いただいているため、対応していきたいと思っています！
